packages/sources/coingecko/src/endpoint/crypto.ts

```diff
@@ -54,6 +72,14 @@ export const inputParameters: InputParameters = {
     description: 'The symbol of the currency to convert to',
     required: true,
   },
+  overrides: {
```

Comment:
These are merged in from the base of https://github.com/smartcontractkit/external-adapters-js/blob/develop/packages/core/bootstrap/src/lib/modules/selector.ts#L15

Reply:
I added symbolToIdOverrides to the baseInputParameters object in lib/modules/selector.ts